### PR TITLE
I2562 - responsibilities and expectations endpoints

### PR DESCRIPTION
**Pull request by weshinsley**
Spec:
https://github.com/vimc/montagu-api/blob/master/docs/spec/Responsibilities.md

Schema:
https://github.com/vimc/montagu-api/blob/master/docs/schemas/Touchstones.schema.json
https://github.com/vimc/montagu-api/blob/master/docs/schemas/ResponsibilitySet.schema.json
https://github.com/vimc/montagu-api/blob/master/docs/schemas/ResponsibilitySet.schema.json
https://github.com/vimc/montagu-api/blob/master/docs/schemas/ResponsibilityDetails.schema.json
https://github.com/vimc/montagu-api/blob/master/docs/schemas/BurdenEstimate.csvschema.json

### CODE REVIEW

**Comment by weshinsley** [comment]
Travis has one note about my use of read.csv in the last function
```
montagu_burden_estimate_template: no visible global function definition
  for ‘read.csv’
Undefined global functions or variables:
  read.csv
Consider adding
  importFrom("utils", "read.csv")
to your NAMESPACE file.
```

**Comment by weshinsley** [comment]
Have also updated orderly dependency to 0.5.14 in DESCRIPTION - to match that in the I2545 PR, which Travis probably won't like yet.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: montagu
 Title: Interface with 'montagu'
-Version: 0.2.5
+Version: 0.2.6
```
Comment (richfitz) [inline DESCRIPTION]: Can I suggest 0.3.0 given the scale of this!

File: R/responsibilities.R [hunk 2]
```diff
@@ -0,0 +1,318 @@
+helper_touchstones <- function(modelling_group_id, location = NULL) {
+  assert_scalar_character(modelling_group_id)
+  path <- sprintf("/modelling-groups/%s/responsibilities/", modelling_group_id)
+  montagu_api_GET(location, path)
+}
+
+##' This may get a name change pending splitting apart of various
```
Comment (richfitz) [inline R/responsibilities.R]: Throughout, you've got one of the stub bits of documentation that are my fault, I think,  Can we replace this with a proper sentence in each case?

File: R/responsibilities.R [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+##' @param touchstone_name Name of the touchstone (ie, no version suffix)
+##' @inheritParams montagu_touchstones
+##' @return Data frame of touchstine id, name, version, description and status
+##' @export
+montagu_touchstone_versions <- function(modelling_group_id,
+                                        touchstone_name, location = NULL) {
+  assert_scalar_character(touchstone_name)
+  res <- helper_touchstones(modelling_group_id, location)
+
+  if (length(res)==0) {
```
Comment (richfitz) [inline R/responsibilities.R]: style wise, could you put spaces around `==` please

File: R/responsibilities.R [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+##' @title Retrieve current status of a groups' scenario.
+##' @param scenario_id Id of the scenario
+##' @inheritParams montagu_scenarios
+##' @return "invalid", "complete", "valid", or "empty"
+##' @export
+montagu_scenario_status <- function(
+  modelling_group_id, touchstone_id, scenario_id, location = NULL) {
+
+  helper_get_responsibility(modelling_group_id, touchstone_id, scenario_id,
+                      location)$status
```
Comment (richfitz) [inline R/responsibilities.R]: indentation - looks like the whole file could benefit from a re-indent

File: DESCRIPTION (hunk 1 above)
Comment (weshinsley) [reply DESCRIPTION]: Done!

File: R/responsibilities.R (hunk 3 above)
Comment (weshinsley) [reply R/responsibilities.R]: Done - always a couple sneak through....

File: R/responsibilities.R (hunk 4 above)
Comment (weshinsley) [reply R/responsibilities.R]: Try now - I've restyled a bit.

Comment (weshinsley) [reply R/responsibilities.R]: The long param and function names sometimes make it a bit messy.

**Comment by weshinsley** [comment]
I've added a vignette beginning the walk through, and documenting all the basic behaviour up to and including this PR. Will extend it on each further PR...

**Comment by weshinsley** [comment]
On the way through writing the vignette, I also came across some naming inconsistencies with the disease lookup, and found that probably a function that says, "just tell me my open touchstones" would be useful, so I re-parameterised one of the functions to do that.

File: R/responsibilities.R (hunk 2 above)
Comment (weshinsley) [reply R/responsibilities.R]: Replaced each with some more descriptive text...

File: vignettes/montagu_user_guide.Rmd [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+To see which outcomes are required for an expectation:
+```{r}
+montagu::montagu_expectation_outcomes("IC-Garske", "201710gavi-5", 30)
+```
+
+And to confirm which scnearios an expectation applies to (which should align
+with the description text for the expectation):
+```{r}
+montagu_expectation_applicable_scenarios("IC-Garske", "201710gavi-5", 30)
```
Comment (richfitz) [inline vignettes/montagu_user_guide.Rmd]: needs a `montagu::`

File: vignettes/montagu_user_guide.Rmd [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+```
+
+### Templates
+
+The above will give all the information required to build your own data frame
+for submitting the expected results. Alternatively, you can download 
+burden estimate templates with the years, countries and ages already populated:-
+
+```{r} 
+df <- montagu_central_burden_estimate_template("IC-Garske", "201710gavi-5", 30)
```
Comment (richfitz) [inline vignettes/montagu_user_guide.Rmd]: also needs a `montagu::`

Comment (weshinsley) [reply vignettes/montagu_user_guide.Rmd]: Corrected

File: vignettes/montagu_user_guide.Rmd (hunk 5 above)
Comment (weshinsley) [reply vignettes/montagu_user_guide.Rmd]: Corrected

**Pull request merged**